Dockerfile

```diff
@@ -1,5 +1,18 @@
 FROM debian:wheezy
-RUN apt-get update -qq && apt-get install -qy python python-pip python-dev git && apt-get clean
+
+RUN apt-get update -qq
+
+# Compose dependencies
+RUN apt-get install -qqy python python-pip python-dev git
+
+# Test dependencies
+RUN apt-get install -qqy apt-transport-https ca-certificates curl lxc iptables
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.3.3 > /usr/local/bin/docker-1.3.3 && chmod +x /usr/local/bin/docker-1.3.3
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.4.1 > /usr/local/bin/docker-1.4.1 && chmod +x /usr/local/bin/docker-1.4.1
+RUN curl https://get.docker.com/builds/Linux/x86_64/docker-1.5.0 > /usr/local/bin/docker-1.5.0 && chmod +x /usr/local/bin/docker-1.5.0
+
+RUN apt-get clean
```

Comment:
This needs to be run on the same line as `apt-get install` otherwise it'll actually make the image _bigger_. (The debs are still in the layers before then this adds another layer that needs downloading.)